src/cfnlint/data/ExtendedSpecs/us-east-1/05_pricing_property_values.json

```diff
@@ -753,93 +760,6 @@
       "cache.t3.small"
     ]
   },
-  {
-    "op": "add",
-    "path": "/ValueTypes/ElasticsearchInstanceType/AllowedValues",
```

Comment:
https://github.com/aws-cloudformation/cfn-lint/pull/1535, bet this switched to being called opensearch in pricing